### ISSUE TITLE
Optimization 

### ISSUE THREAD

**Issue by jianswang** [automl/auto-sklearn#638]
I do understand that loss in abstract_evaluator.py is used for optimizer. However, I realised that there are also train_loss and test_loss. Are they used for optimizer? Or there are other variables involved in the optimizer?

**Comment by mfeurer** [comment]
No, `train_loss` and `test_loss` are only computed to inspect them after the training procedure. Only the validation loss `loss` is used during optimization.
